anxcloud/data_source_tags.go

```diff
@@ -38,10 +40,8 @@ func dataSourceTagsRead(ctx context.Context, d *schema.ResourceData, m interface
 		return diag.FromErr(err)
 	}
 
-	if id := uuid.New().String(); id != "" {
-		d.SetId(id)
-		return nil
-	}
-
-	return diag.Errorf("unable to create uuid for tags data source")
+	id := fmt.Sprintf("%s-%s-%s-%s",
```

Comment:
Could you check if query, serviceid and organization id are empty? I would like to avoid having an id like 123234---

Reply:
done